src/modules/organization/organization.service.ts

```diff
@@ -87,10 +87,20 @@ export class OrganizationService {
    * returns single Org with matching namespace
    * @param {String} owner
```

Comment:
Please update params documentation

Reply:
@JGiter We still need to describe the `withRoles` and `withApps` param. Is this description correct?
```suggestion
   * @param {String} owner
   * @param {object} [object.withRoles] - Include role data in query result
   * @param {object} [object.withApps] - Include app data in query result
```